styleguide/derek/_partials/solutions/announcements.ejs

```diff
@@ -0,0 +1,53 @@
+<div class="container-fluid standard-padding bg-light-gray">
+  <div class="container">
+
+    <div class="center-block">
+      <h2 class="center teal">Announcements</h2>
```

Comment:
There is a `center-block()` mixin that can be utilized here to allow for maintainable css classes